.github/workflows/checks.yml

```
@@ -169,7 +169,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: ["1.22.11", "1.23.1"] # 1.20 is interpreted as 1.2 without quotes
+        go-version: ["1.22.12", "1.23.1"] # 1.20 is interpreted as 1.2 without quotes
```

Use the latest patched version of this minor version:


```suggestion
        go-version: ["1.22.12", "1.23.6"] # 1.20 is interpreted as 1.2 without quotes
```